src/main/java/TechJobs.java

```diff
@@ -120,6 +120,20 @@ private static String getUserSelection(String menuHeader, HashMap<String, String
     // Print a list of jobs
     private static void printJobs(ArrayList<HashMap<String, String>> someJobs) {
 
-        System.out.println("printJobs is not implemented yet");
+        for (HashMap<String, String> jobs: someJobs) {
+            System.out.println("*****");
```

Comment:
same comment here about the println, the added spaces are causing tests to fail. Use System.out.print() instead